docs/manuals/gui/styling/stylekit.md

```diff
@@ -0,0 +1,349 @@
+# The Stylekit
+
+Taipy GUI comes with predefined cascading stylesheets that solidify your application's
+general look and feel. Those stylesheets also make it far easier to customize styles.</br>
+All those stylesheets are grouped in what is called the *Stylekit*.
+
+The Stylekit provides a few useful features:
+
+- Predefined styles for Taipy GUI visual elements so they align better on the page;
+- CSS variables that are used all over the stylesheets so that you can modify those
+  variables and witness the propagation of specific graphical attributes;
+- Utility CSS classes that can be used to make some elements appear with appealing
+  attributes.
+
+## Using the Stylekit
+
+The Stylekit can apply to your pages if you choose and you can customize most of its properties.
+
+### Enabling or disabling the Stylekit
+
+By default, the Stylekit is enabled: the Stylekit stylesheets are loaded with your application
+and impact all your application's page rendering.
+
+In order *not* to use the Stylekit, you must specify the *stylekit* of the
+`Gui.run()^` method to False:
+```py
+gui.run(stylekit=False)
+```
+
+Note that setting *stylekit* to True is equivalent to using the default variable values of
+the Stylekit.
+
+### Customizing the Stylekit
+
+The Stylekit defines a set of [variables](#variables) with default values. Those variables can be
+overloaded by setting the *stylekit* parameter of the `Gui.run()^` method to a dictionary:
+each key of this dictionary represents a Stylekit variable name, and you would associate the new value
+for this variable to that key.
+
+Here is how you can change the primary and secondary colors for your application:
+
+```py
+stylekit = {
+  "color_primary": "#BADA55",
+  "color_secondary": "#C0FFE",
+}
+gui.run(stylekit=stylekit)
+```
+
+The exhaustive list of all the Stylekit variables can be found in the
+[Variables](#variables) section below.
+
+## Variables
+
+The Stylekit defines several variables that impact the overall style of your Taipy GUI
+application pages. The values for these variables can be overloaded by providing a dictionary
+to the *stylekit* parameter of the `Gui.run()^` method.
+
+Each variable initializes a CSS custom property that you can use in your own stylesheets.<br/>
```

Comment:
And modify the Mui base theme

Reply:
Although this is true, I'm trying to hide those implementation details from 'regular' users.
May be I could create a more technical document in taipy-gui/gui to expose those kind of things.
Thoughts?

Reply:
that's a plan